### DISCUSSION TITLE
更新最新的v3.37.4版本之后，首页加载非常非常慢

### DISCUSSION THREAD

**Discussion by zhlx2835** [AlistGo/alist#7230] [Q&A]
如题，更新最新的v3.37.4版本之后，首页加载非常非常慢。
从访问链接里看到，一些 assets/index.51db64c5.js加载用了29s，其他几个 js 访问也非常慢，
想问下，这几个 js 有什么用，怎么优化。
![alist慢](https://github.com/user-attachments/assets/f775308f-182b-499c-8b62-906171600c20)

**Comment by zhlx2835** [comment]
而且进入后台的存储加载也很慢，没升级之前都正常的。

**Comment by zhlx2835** [comment]
哪个大佬知道是怎么回事？
有没有遇到相同问题的

**Comment by anwen-anyi** [comment]
去你的网站看来下，网络问题吧？ 我去试了下 Ctrl+F5 刷新清除缓存第一次访问没那么慢吖
![image](https://github.com/user-attachments/assets/d6faf4c1-c4b9-41cb-86e2-ef401364b896)

**Comment by starskyzheng** [comment]
同样的情况 v3.41.0
![image](https://github.com/user-attachments/assets/38b74deb-a802-4ebb-a3e3-149c99a108f2)
